src/main/resources/application.yml

```diff
@@ -23,6 +23,9 @@ application:
     headers:
       Content-Type: "application/json"
   maxStepRecursions: 10
+  dslParameters:
+    DOMAIN_URL: "https://example.com"
+    PORT: 8080
```

Comment:
Just to clarify - `DOMAIN_URL` and `PORT` are just sample keys, right? There can be any nr of them and all of them can be used in DSLs?

Reply:
Correct.